src/hotspot/share/jfr/instrumentation/jfrEventClassTransformer.cpp

```diff
@@ -1536,27 +1536,27 @@ static ClassFileStream* retransform_bytes(const Klass* existing_klass, const Cla
                               THREAD);
   JfrJavaSupport::destroy_local_jni_handle(clazz);
   if (has_pending_exception(THREAD)) {
-    return NULL;
+    return nullptr;
   }
-  assert(new_bytes != NULL, "invariant");
+  assert(new_bytes != nullptr, "invariant");
   assert(size_of_new_bytes > 0, "invariant");
   is_instrumented = true;
-  return new ClassFileStream(new_bytes, size_of_new_bytes, NULL, ClassFileStream::verify);
+  return new ClassFileStream(new_bytes, size_of_new_bytes, nullptr, ClassFileStream::verify);
 }
 
 // On initial class load.
 static void cache_class_file_data(InstanceKlass* new_ik, const ClassFileStream* new_stream, const JavaThread* thread) {
-  assert(new_ik != NULL, "invariant");
-  assert(new_stream != NULL, "invariant");
-  assert(thread != NULL, "invariant");
+  assert(new_ik != nullptr, "invariant");
+  assert(new_stream != nullptr, "invariant");
+  assert(thread != nullptr, "invariant");
   assert(!thread->has_pending_exception(), "invariant");
   if (!JfrOptionSet::allow_retransforms()) {
     return;
   }
   const jint stream_len = new_stream->length();
   JvmtiCachedClassFileData* p =
-    (JvmtiCachedClassFileData*)NEW_C_HEAP_ARRAY_RETURN_NULL(u1, offset_of(JvmtiCachedClassFileData, data) + stream_len, mtInternal);
-  if (p == NULL) {
+    (JvmtiCachedClassFileData*)NEW_C_HEAP_ARRAY_RETURN_nullptr(u1, offset_of(JvmtiCachedClassFileData, data) + stream_len, mtInternal);
```

Comment:
fix

Reply:
Not sure agree about the new macro. Why not in capitals, if it needs to be updated (which I am not sure it has)?